### ISSUE TITLE
Could not find method classpath() for arguments

### ISSUE THREAD

**Issue by vtomole** [bintray/gradle-bintray-plugin#247]
I added 
```
buildscript {
    repositories {
        jcenter()
    }
    dependencies {
        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.0'
    }
}
```

to my build.gradle and when I `./gradlew build`, I get 

```
vtomole@vtomole:~/jQuil$ ./gradlew build

FAILURE: Build failed with an exception.

* Where:
Build file '/home/vtomole/jQuil/build.gradle' line: 23

* What went wrong:
A problem occurred evaluating root project 'jQuil'.
> Could not find method classpath() for arguments [com.jfrog.bintray.gradle:gradle-bintray-plugin:1.+] on object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 0s
vtomole@vtomole:~/jQuil$ ./gradlew build

FAILURE: Build failed with an exception.

* Where:
Build file '/home/vtomole/jQuil/build.gradle' line: 23

* What went wrong:
A problem occurred evaluating root project 'jQuil'.
> Could not find method classpath() for arguments [com.jfrog.bintray.gradle:gradle-bintray-plugin:1.+] on object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
```

How do I fix this?

Thank you.

**Comment by vtomole** [comment]
Instead of implementing my own, I just copied from here: https://github.com/bintray/bintray-examples

**Comment by Abdullah2500** [comment]
I still didn't understand how you solved the issue?

**Comment by vtomole** [comment]
Hey @Abdullah2500,

This issue was a long time ago so I don't remember exactly how I solved it. I ended up completing the project that this required; and that project's `build.gradle` is here: https://github.com/QCHackers/jquil/blob/master/build.gradle.

This was probbably solved by adding ` apply plugin: 'com.jfrog.bintray'`: https://github.com/QCHackers/jquil/blob/3493e2a69665a0b3a37bc07ab34d9095ecfce493/build.gradle#L20.

Let me know if that works.
